#92 [phpcs] Apply coding standard changes

pritalpatel opened the pull request:
Thanks @yvesh for suggestions.
Send Coding standard changes for https://github.com/joomla-projects/joomla-browser/issues/92

yvesh commented on src/JoomlaBrowser.php:
```diff
@@ -110,191 +128,212 @@ public function doFrontendLogout()
+		else :
+		    $this->debug('I install Joomla without Sample Data');
+		    $I->selectOption('#jform_sample_file', '#jform_sample_file0'); // No sample data
+		endif;*/
```
Don't leave any commented code.

https://medium.com/@kentcdodds/please-don-t-commit-commented-out-code-53d0b5b26d5f#.n7t3shymb

pritalpatel replied:
I leave it because it says `@todo` there. I will remove it.

yvesh commented on src/JoomlaBrowser.php:
```diff
@@ -886,15 +970,17 @@ public function verifyAvailableTabs($expectedTabs, $tabsLocator = ['xpath' => "/
+		$I->waitForElement(['link' => 'Never'], 60);
+		$I->click(['link' => 'Never']);
     }
 }
```
Add a newline after the closing }

yvesh commented:
Hi @pritalpatel, your PR has one big problem. Your tab is 8 spaces long and not 4. Can you adjust that please? Thank you :)

pritalpatel commented:
Thanks @yvesh Actually that problem with 8 char was because of I have fixed conflict with meld tool. Now, I have corrected it from Sublime Text. So, should be fine now.

javigomez commented:
closing for #116
